contracts/contracts/core/PreconfManager.sol

```diff
@@ -83,6 +83,8 @@ contract PreconfManager is
 
         // Compute the domain separators
         uint256 chainId;
+        
+        // solhint-disable-next-line no-inline-assembly
```

Comment:
Can remove this line, I changed it to `block.chanid` in other PR

Reply:
Great!